### PR TITLE
Android

### PR DESCRIPTION
**Pull request by evelyn323**
Android changes

### CODE REVIEW

**Review by ddxxdd-code — COMMENTED** [review]
All about refactoring of file structure

**Review by dabandaidai — CHANGES_REQUESTED** [review]
setContentView(R.layout.staff) needs to be changed.

File: Android/app/src/main/java/com/example/androidgui/ServeDishActivity.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import android.widget.TextView;
+import androidx.appcompat.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class ServeDishActivity extends AppCompatActivity {
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.staff);
+        TextView header = (TextView) findViewById(R.id.header);
```
Comment (evelyn323) [inline Android/app/src/main/java/com/example/androidgui/ServeDishActivity.java:12]: Cast to TextView is redundant, remove cast

File: Android/app/src/main/java/com/example/androidgui/ServeDishActivity.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        header.setText("Serve Dish System");
```
Comment (evelyn323) [inline Android/app/src/main/java/com/example/androidgui/ServeDishActivity.java:13]: Don't use string literal, use Android resources instead

File: Android/app/src/main/java/com/example/androidgui/DeliverOrderActivity.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import android.widget.TextView;
+import androidx.appcompat.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class DeliverOrderActivity extends AppCompatActivity {
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.staff);
+        TextView header = (TextView) findViewById(R.id.header);
```
Comment (evelyn323) [inline Android/app/src/main/java/com/example/androidgui/DeliverOrderActivity.java:12]: Cast to TextView is redundant, remove cast

File: Android/app/src/main/java/com/example/androidgui/DeliverOrderActivity.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+        header.setText("Serve Dish System");
```
Comment (evelyn323) [inline Android/app/src/main/java/com/example/androidgui/DeliverOrderActivity.java:13]: Don't use string literal, use Android resources instead

**Review by dabandaidai — COMMENTED** [review]
1. Namings need to be better: dn, qt, bt in KitchenAdapter lines 35-37 not appropriate enough.
2. Adequate use of constants in cook_dish_layout.
3. Redundant/useless code in KitchenController line 153-157.
4. Same problem as 3 in KitchenActivity line 33-41.

**Review by dabandaidai — APPROVED** [review]
More comments needed. 
Not much problem, can be approved.

**Review by dabandaidai — APPROVED** [review]
Lots of TODOs left.
More comments would be appropriate.

**Review by dabandaidai — APPROVED** [review]
Edited the file paths.
Deleted TODOs(are they really unnecessary?)

**Review by dabandaidai — APPROVED** [review]
Update of PlaceOrderActivity to SeeMenuActivity, required in the specifications.
Also the method selectPlaceOrder to selectSeeMenu.

File: Android/app/src/main/java/com/example/androidgui/DeliverOrderActivity.java (hunk 4 above)
Comment (ddxxdd-code) [reply Android/app/src/main/java/com/example/androidgui/DeliverOrderActivity.java]: Resolved in commit 44254f77c4d79245bf4b70fe7f702b8f1a71dc3f

File: Android/app/src/main/java/com/example/androidgui/DeliverOrderActivity.java (hunk 3 above)
Comment (ddxxdd-code) [reply Android/app/src/main/java/com/example/androidgui/DeliverOrderActivity.java]: Resolved in commit 44254f77c4d79245bf4b70fe7f702b8f1a71dc3f

**Review by dabandaidai — APPROVED** [review]
Switch from hellodroid to androidgui.
Changed names of several activities, related classes and methods.

**Review by dabandaidai — APPROVED** [review]
Fixing some simple errors.
Not much to comment.

**Review by dabandaidai — APPROVED** [review]
Simple error fixing changes.
Not much to comment.

File: Android/app/src/main/java/com/example/androidgui/ServeDishActivity.java (hunk 2 above)
Comment (ddxxdd-code) [reply Android/app/src/main/java/com/example/androidgui/ServeDishActivity.java]: Resolved in commit 44254f77c4d79245bf4b70fe7f702b8f1a71dc3f

File: Android/app/src/main/java/com/example/androidgui/ServeDishActivity.java (hunk 1 above)
Comment (ddxxdd-code) [reply Android/app/src/main/java/com/example/androidgui/ServeDishActivity.java]: Resolved in commit 44254f77c4d79245bf4b70fe7f702b8f1a71dc3f

**Review by dabandaidai — APPROVED** [review]
Transformation from original main to the Android project model. 
Included change of file location and names.

**Review by dabandaidai — APPROVED** [review]
Transformation from original main class under an Android mode.

**Review by dabandaidai — APPROVED** [review]
Not much to comment.
Mostly file name changed and file path changes.

**Review by dabandaidai — APPROVED** [review]

**Pull request closed without merging**